refactor(examples): modularize account_create_transaction_create_with_alias

adityashirsatrao007 opened the pull request:
## Description

This PR refactors the \ccount_create_transaction_create_with_alias.py\ example to improve readability and maintainability.

### Changes

- Split the monolithic \create_account_with_separate_ecdsa_alias()\ function into focused, modular functions:
  - \generate_main_and_alias_keys()\ - Generates both the main account key and the alias ECDSA key
  - \create_account_with_ecdsa_alias()\ - Creates the account with the alias configuration
  - \etch_account_info()\ - Fetches account information from the network
  - \print_account_summary()\ - Displays the account details in a formatted way
- Updated \main()\ to orchestrate calls to these modular functions
- Added CHANGELOG entry

### Related Issue

Closes #1016

### Checklist

- [x] Code follows the project's coding standards
- [x] Example is functional and demonstrates the intended use case
- [x] CHANGELOG updated

Signed-off-by: Aditya Shirsatrao <adityashirsatrao@gmail.com>

AntonioCeppellini commented on examples/account/account_create_transaction_create_with_alias.py:
```diff
@@ -49,83 +50,132 @@ def setup_client():
+    """
+    print("\nSTEP 2: Creating the account with the EVM alias from the ECDSA key...")
+
+    alias_public_key = alias_private_key.public_key()
```
you created it before in `def generate_main_and_alias_keys()`

adityashirsatrao007 replied:
Fixed. I renamed the function to `generate_main_and_alias_keys` and ensured keys are generated there.

AntonioCeppellini commented on examples/account/account_create_transaction_create_with_alias.py:
```diff
@@ -49,83 +50,132 @@ def setup_client():
     """Main entry point."""
-    client = setup_client()
-    create_account_with_separate_ecdsa_alias(client)
+    try:
```
maybe we can use the `try ... except` inside the functions and not in the main in order to stop the example when it breaks, what do you think @exploreriii ?

adityashirsatrao007 commented:
I have refactored the example to modularize the code and added try-except blocks as requested. Renamed the key generation function to 'generate_main_and_alias_keys'.
